naucse/templates/_base.html

```diff
@@ -49,12 +49,19 @@
         <div class="footer container">
             <hr>
             <div class="lesson-attribution">
-                <p>
+                {% if edit_info is defined and edit_info %} {# the page is from fork #}
```

Comment:
Instead of not setting edit_info on non forks, make it so the default values are the ones that would result in the following else in this template. Than, you cen get rid of this if-elsing entirely.